INSTALL.md

```diff
@@ -1,11 +1,47 @@
+## Installation requirements on ubuntu
+
 WCIVF requires Python 3.
 
 To install:
 
     sudo apt-get install python3-dev libpq-dev libjpeg-dev redis-server
     pip install -r requirements/local.txt
 
-Create a Postgres database as detailed below, then:
+
+## Installation requirements on macOS
+
+Use homebrew to install the following dependencies:
+
+    brew install postgresql libjpeg-dev postgis
+
+Pyenv is recommended to install python versions. You can find installation instructions for pyenv [here](https://github.com/pyenv/pyenv#installation) - installation via homebrew is recommended. You can then install python 3 with:
+
+    pyenv install 3.6.12
+
+If using pyenv, after installation use the command `pyenv local 3.6.12` to set this version for the project locally. This will create a `.python-version` file with the specified python version.
+
+You can then create a [virtual environment](https://docs.python.org/3/tutorial/venv.html) with the command:
+
+    python -m venv env
```

Comment:
This command didn't work for me but `python3 -m venv env` did.

Reply:
This is strange...is `python` found at all? I'd have thought that `pyenv` would have put both `python` and `python3` on your PATH. What does `python --version` show?

Reply:
I think this may have been something to do with the pyenv installation. Now that I have commited the `.python-version` file hopefully this should help with this. Whenever there is a `.python-version` file present in a directory, Pyenv will attempt to activate and use that version. If the specified is not installed, you'd get an error when running `python` command like:

```
michaelcollins@Michaels-MBP WhoCanIVoteFor % python -V                          
pyenv: version `3.6.1' is not installed (set by /Users/michaelcollins/code/dc/WhoCanIVoteFor/.python-version)
```

More info here https://realpython.com/intro-to-pyenv/

Reply:
FWIW here is a step by step shell output of me 
- opening a new terminal session 
- displaying the default python version as system 
- then changing to the WhoCanIVoteFor dir and displaying the 3.6.12 version being used
- then creating the venv, activating it and again checking the version

Might be useful to reference against steps you are taking
```
michaelcollins@Michaels-MBP ~ % pwd
/Users/michaelcollins
michaelcollins@Michaels-MBP ~ % python -V
Python 2.7.16
michaelcollins@Michaels-MBP ~ % cd code/dc/WhoCanIVoteFor 
michaelcollins@Michaels-MBP WhoCanIVoteFor % python -V
Python 3.6.12
michaelcollins@Michaels-MBP WhoCanIVoteFor % pyenv versions
  system
* 3.6.12 (set by /Users/michaelcollins/code/dc/WhoCanIVoteFor/.python-version)
  3.7.9
  3.8.6
  3.9.0
michaelcollins@Michaels-MBP WhoCanIVoteFor % python -m venv env
michaelcollins@Michaels-MBP WhoCanIVoteFor % source env/bin/activate
(env) michaelcollins@Michaels-MBP WhoCanIVoteFor % python -V
Python 3.6.12
(env) michaelcollins@Michaels-MBP WhoCanIVoteFor % 
```

Reply:
Very helpful, thank you! These steps worked for me. 👍